sly.el

```diff
@@ -5331,8 +5370,7 @@ pending Emacs continuations."
                  t)
                () "Bug: sly-db-level is equal but condition differs\n%s\n%s"
                sly-db-condition condition)
-    (pop-to-buffer (current-buffer) '(sly-db--display-in-prev-sly-db-window))
-    (set-window-parameter (selected-window) 'sly-db (current-buffer))
+    (sly-db-display-debugger (current-buffer))
```

Comment:
If your `sly-db-display-debugger` function (which you should rename to `sly-db--display-debugger`) is guaranteed to always return a window, you can probably use it as the `with-selected-window` argument, i.e. move that line upwards from 5397 to 5373.

Reply:
I tried moving `with-selected-window`, but it changes how the backtrace is shown: given a sufficiently long error, the window has the first line of the backtrace centered, instead of the backtrace beginning near the bottom of the window (as it does now).